### PR TITLE
Test RSpec

### PR DESCRIPTION
**Pull request by wandji20**
In this project, We have made valid test examples for enumerable methods with RSpec.

### CODE REVIEW

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle:
### Hi @wandji20 and @elisha2kyakpo1 ! 👋 👋 👋 

#### You have done well so far :clap: However, there are changes required for you to make in order for this project to be approved. Kindly work on them and submit them for another code review.

**Recommended**

- [x] You have done well so far, all you have to do now is fix your rubocop errors. `rubocop -A` can fix some of them.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

File: spec/our_rspec_test.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+require_relative '../ruby'
+
+describe Enumerable do
+  describe '#my_each' do
+    it 'Performs a block of code if given and returns the passed enumerable' do
+      expect([1, 2, 3].my_each { |x| 5 * x }).to eq([1, 2, 3])
+    end
+  end
```
Comment (peterrobert) [inline spec/our_rspec_test.rb]: Great work adding this test. However, you have only tested for a positive scenario. What happens if no block is given? You should also test for the negative scenarios.

commonsense (1)

[pathwright link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/)

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED ♻️ 🛑 
HELLO @wandji20, 👋🏿  VERY NICE WORK 👍🏿 YOU HAVE DONE SUCH A IMPRESSIVE JOB. I LOVE THE WAY YOU HAVE CREATED TESTS FOR ALL YOUR METHODS 👍🏿 . HOWEVER, THERE ARE A FEW ISSUES THAT NEED TO BE ADDRESSED KINDLY CHECK BELOW ⬇️ 

- [x] You have tested all of your methods 👍🏿 but you have only tested for positive scenarios. kindly remember you need to test for both positive and negative scenarios kindly check below for a hint.

### if you have any question kindly reach me on slack @peterrobert
### HAPPY CODING 💻

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Elisha Kyakopo & Wandji Wandji** 
Nice job with the improvements for this project :muscle:  :muscle:
Your tests met all the requirements :clap: :clap: :clap:  
Just a final detail to improve before you can go ahead:

- [x] In the README file add a screenshot and explain how to run the tests
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by ghost — APPROVED** [review]
# Status: Approved 🎉 

**Congratulations team and good luck with your next project 😃.**
